number_guesser/number_guess.py

```diff
@@ -0,0 +1,32 @@
+from random import randint
+
+LN = 50  # LN - largest number of range
+n = randint(0, LN)
+
+# TODO: computer can generate 0 also
+answer = str(input("I thought of a number from 0 to " + str(LN) + ". Do you want to guess?\n"))
+if 'yes' in answer or 'YES' in answer or 'Yes' in answer:
+    print('OK. Input your number:')
```

Comment:
there is no input, the program has print only

Reply:
Made a change